Yoast/wordpress-seo
# Move active marker id to store

## Summary

This PR can be summarized in the following changelog entry:

* Fixes a bug where multiple marker buttons could be active at the same time.

## Relevant technical choices:

* Also backports `@wordpress/compose` just like we did with a few other wp dependencies.

## Test instructions

This PR can be tested by following these steps:

* Check out this branch and link the `11572-active-marker-status-in-store` branch from `yoast-components` to `wordpress-seo`. https://github.com/Yoast/yoast-components/pull/776
* Open a post, and create a post that is long enough to hold multiple keyphrases. Use [Lorem ipsum](https://nl.lipsum.com/feed/html) and use the keyprase `lorem` if necessary.
* Open the focus keyphrase collapsible in the metabox, click the marker next to the `Keyphrase density` assessment.
* Collapse and reopen the focus keyphrase collapsible and make sure the marker button AND the markers are still active.
* Check if you can deactivate the marker.
* NB. related keywords in premium don't have marker buttons

## Quality assurance


Fixes #11572 
Fixes #10618
